cdk/lib/braze-components.ts

```diff
@@ -20,11 +21,16 @@ export class BrazeComponents extends GuStack {
     constructor(scope: App, id: string, props: BrazeComponentsStackProps) {
         super(scope, id, props);
 
-        const bucketNameFromStaticCloudformationStack = 'braze-components-storybook';
+        // Get bucket name from SSM Parameter Store
```

Comment:
I think it should be self explanatory and comments often end up outdated or separated from their associated code
```suggestion
```